Corona-Warn-App/src/main/java/de/rki/coronawarnapp/environment/EnvironmentSetup.kt

```diff
@@ -36,6 +38,8 @@ class EnvironmentSetup @Inject constructor(
         VERIFICATION_KEYS("PUB_KEYS_SIGNATURE_VERIFICATION"),
         DATA_DONATION("DATA_DONATION_CDN_URL"),
         LOG_UPLOAD("LOG_UPLOAD_SERVER_URL"),
+        VACCINATION_PROOF("VACCINATION_PROOF_SERVER_URL"),
+        VACCINATION_VALUE("VACCINATION_VALUE_SET_CDN_URL"),
```

Comment:
Let's pick a different name here, as "value sets" is just a specific data set that can be returned here, not sure what we could call it though, maybe just VACCINATION_CDN? Does the tech spec call it something useful?